src/helm-frontend/src/services/getDisplayPredictionsByName.ts

```diff
@@ -1,20 +1,32 @@
 import type DisplayPrediction from "@/types/DisplayPrediction";
 import getBenchmarkEndpoint from "@/utils/getBenchmarkEndpoint";
-import getVersionBaseUrl from "@/utils/getVersionBaseUrl";
+import getBenchmarkSuite from "@/utils/getBenchmarkSuite";
 
 export default async function getDisplayPredictionsByName(
   runName: string,
   signal: AbortSignal,
+  suite?: string,
 ): Promise<DisplayPrediction[]> {
   try {
-    const displayPrediction = await fetch(
-      getBenchmarkEndpoint(
-        `${getVersionBaseUrl()}/${runName}/display_predictions.json`,
-      ),
-      { signal },
-    );
+    if (suite) {
+      const displayPrediction = await fetch(
+        getBenchmarkEndpoint(
+          `/benchmark_output/runs/${suite}/${runName}/display_predictions.json`,
```

Comment:
For all of these, you could do
```
`/benchmark_output/runs/${suite || getBenchmarkSuite()}/${runName}/display_predictions.json`,
```

Then you can delete the `if` and just have one branch.

Reply:
Thanks, this is much cleaner, fixed